static/app/components/avatarChooser.tsx

```diff
@@ -130,14 +167,21 @@ class AvatarChooser extends React.Component<Props, State> {
     if (!model) {
       return <LoadingIndicator />;
     }
+    const {allowDefault, preview, choiceText: defaultChoiceText} = defaultChoice || {};
 
     const avatarType = model.avatar?.avatarType ?? 'letter_avatar';
     const isLetter = avatarType === 'letter_avatar';
+    const isDefault = preview !== undefined && avatarType === 'default';
```

Comment:
NIT: 
```suggestion
    const isDefault = preview && avatarType === 'default';
```

Reply:
Fixed! wrapped in `Boolean()` to fix its typing as well.